packages/bueno/esbuild.js

```diff
@@ -0,0 +1,52 @@
+const {build} = require('esbuild');
+const {apacheLicense} = require('../../scripts/license/apache');
+
+const devMode = process.argv[2] === 'dev';
+
+/**
+ * @type {import('esbuild').BuildOptions}
+ */
+const base = {
+  entryPoints: ['src/index.ts'],
+  bundle: true,
+  banner: {js: apacheLicense()},
+  define: {
+    'process.env.NODE_ENV': JSON.stringify('production'),
+  },
+};
+
+function nodeCjs() {
+  return build({
+    ...base,
+    platform: 'node',
+    outfile: 'dist/bueno.js',
+    format: 'cjs',
+  });
+}
+
+function nodeEsm() {
+  return build({
+    ...base,
+    platform: 'node',
+    outfile: 'dist/bueno.esm.js',
+    format: 'esm',
+  });
+}
+
+function browserEsm() {
+  return build({
+    ...base,
+    platform: 'browser',
+    outfile: 'dist/browser/bueno.esm.js',
+    format: 'esm',
+    watch: devMode,
+    minify: true,
+    sourcemap: true,
```

Comment:
Since bueno is not served on a CDN, rather consumed via a bundler, in theory these minification options should not be needed (consumer can choose whether or not to minify). I kept them to minimize differences with the bundle generated by rollup, but I wouldn't mind removing them.

Thoughts?

Reply:
You can remove them